trade/trader_stakecube.js

```diff
@@ -582,19 +600,21 @@ module.exports = (
       const paramString = `pair: ${pair}`;
       const pair_ = formatPairName(pair);
 
-      let trades = [];
+      let scData = [];
       try {
-        trades = await stakeCubeApiClient.getTradesHistory(pair_.pairPlain, limit);
+        scData = await stakeCubeApiClient.getTradesHistory(pair_.pairPlain, limit);
 
-        if (trades.errorMessage) {
-          log.warn(`API request getTradesHistory(${paramString}) of ${utils.getModuleName(module.id)} module failed. ${trades.errorMessage}}`);
+        if (scData.error) {
```

Comment:
It's not necessary to check this as in case of error, function will be try-catched.
In case of error, we already logged it in `stakecube_api`.